wazo_amid/ami/tests/test_client.py

```diff
@@ -15,27 +18,32 @@
 
 from wazo_amid.ami.client import AMIClient, AMIConnectionError
 
+if TYPE_CHECKING:
+    from typing import ParamSpec, TYPE_CHECKING
```

Comment:
Is not redundant to import TYPE_CHECKING in a clause using TYPE_CHECKING? :smile:

Reply:
Quite! This is why we should have tests. 😅